assets/js/data/cart/actions.js

```diff
@@ -119,3 +119,59 @@ export function* removeCoupon( couponCode ) {
 
 	yield receiveRemovingCoupon( '' );
 }
+
+/**
+ * Returns an action object to indicate if the specified cart item
+ * is being updated; i.e. removing, or changing quantity.
```

Comment:
Nice idea about given this a more broad purpose for indicating pending state for quantity selector with removal and other updates 👏